src/core/Akka.API.Tests/CoreAPISpec.ApprovePersistence.approved.txt

```diff
@@ -854,7 +854,7 @@ namespace Akka.Persistence.Fsm
             public System.Action<TData> AfterTransitionHandler { get; }
             public Akka.Util.ILinearSeq<TEvent> DomainEvents { get; }
             public bool Notifies { get; set; }
-            public System.Collections.Generic.IReadOnlyCollection<object> Replies { get; }
+            public System.Collections.Generic.IReadOnlyList<object> Replies { get; }
```

Comment:
Because I implemented this code :)
```csharp
for (int i = nextState.Replies.Count - 1; i >= 0; i--)
{
    Sender.Tell(nextState.Replies[i]);
}
```